### PR TITLE
[EventGrid] Regen, remove msrest and six

### PR DESCRIPTION
**Pull request by l0lawrence**
Removing msrest and six dependency and regenerating

### CODE REVIEW

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from typing import Any, cast
 import datetime as dt
 import uuid
-from msrest.serialization import UTC
+from datetime import datetime
```
Comment (kashifkhan) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py:9]: datetime is already being imported above on line 7, can be combined

File: sdk/eventgrid/azure-eventgrid/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,10 @@
 
 - `SystemEventNames` enums for APIManagement were incorrectly capitalized, changed `Api` to `API`.
 
+### Other Changes
+
+- Removed `msrest` dependency and `six` references, and added `isodate` dependency
```
Comment (kashifkhan) [inline sdk/eventgrid/azure-eventgrid/CHANGELOG.md:15]: nit
```suggestion
- Removed `msrest` dependency and `six` dependencies
- Added `isodate` dependency
```

File: sdk/eventgrid/azure-eventgrid/swagger/README.PYTHON_T2.md [hunk 3]
```diff
@@ -10,15 +10,11 @@ description: EventGrid Python Publisher Client
 generated-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 no-namespace-folders: true
-low-level-client: true
-show-operations: false
-models: msrest
 output-folder: ../azure/eventgrid/_generated
-source-code-folder-path: ./azure/eventgrid/_generated
+source-code-folder-path: ./azure/eventgrid/_new_generated_code
```
Comment (swathipil) [inline sdk/eventgrid/azure-eventgrid/swagger/README.PYTHON_T2.md:14]: I think you may want to update this back to the original?
```suggestion
source-code-folder-path: ./azure/eventgrid/_generated
```

**Review by swathipil — APPROVED** [review]

**Review by kashifkhan — APPROVED** [review]

File: sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py [hunk 4]
```diff
@@ -87,7 +86,7 @@ def __init__(self, subject, event_type, data, data_version, **kwargs):
         kwargs.setdefault("id", uuid.uuid4())
         kwargs.setdefault("subject", subject)
         kwargs.setdefault("event_type", event_type)
-        kwargs.setdefault("event_time", dt.datetime.now(UTC()).isoformat())
+        kwargs.setdefault("event_time", dt.datetime.utcnow().isoformat())
```
Comment (l0lawrence) [inline sdk/eventgrid/azure-eventgrid/azure/eventgrid/_models.py:89]: Apparently utcnow() doesn't deal with timezones?

**Review by lmazuel — APPROVED** [review]
I checked everything outside of the _generated folder, as this is manual code, and it feels all good

**Pull request merged**